File: PHP/ComprasNoSupermercado.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

//echo "The result is "+checkMyList(["laranja", "pera"]);

class ComprasNoSupermercado {

    /**
     * Remove duplicates and organize my list.
     */
    public function checkMyList($items) {
        $result = array();

        foreach ($items as &$item) {
            if (!in_array($item, $result, true)) {
                array_push($result, $item);
            }
        }
    
        sort($result);
        return $result;
    }
}

File: GO/ComprasNoSupermercado.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import (
	"sort"
)

/**
 * Remove duplicates and organize my list.
 */
func checkMyList(items []string) []string {
	var result []string

	for _, item := range items {
		var contem bool = false

		for _, r := range result {
			if r == item {
				contem = true
				break
			}
		}

		if !contem {
			result = append(result, item)
		}
	}

	sort.Strings(result)
	return result
}
